airflow/operators/docker_operator.py

```diff
@@ -198,7 +198,7 @@ def execute(self, context):
 
         if self.force_pull or len(self.cli.images(name=self.image)) == 0:
```

Comment:
Do you need `len(self.cli.images(name=self.image)) == 0` check? That is I think is default behavior.